readme/simplesamlphp-setup.md

```diff
@@ -0,0 +1,46 @@
+# SimpleSAMLphp Setup
```

Comment:
Please add this file to the mkdocs.yml navigation tree and reference this document in project-tasks.md.